### ISSUE TITLE
p:dataTable: Frozen columns row Height not synced with data row height

### ISSUE THREAD

**Issue by wlhUser** [primefaces/primefaces#5004]
## 1) Environment
- PrimeFaces version: 7.0.5

- Application server + version: WildFly 15
- Affected browsers: 
All

## 2) Expected behavior
With a p:dataTable with frozen Columns, I expect that the row heights of the frozen columns and the data columns is synchronized, as otherwise rows belonging together are not displayed aligned. 
This does not happen in standard primeFaces, but in case the styleSheets had been changed in order to show smaller row heights
![grafik](https://user-images.githubusercontent.com/47218021/61459469-1035c700-a96d-11e9-9452-52845ab04ded.png)

## 4) Steps to reproduce
Please have a look at the example I've added as zip file.
It contains:
XHTML
Bean
CSS
theme

[PF_ticket.zip](https://github.com/primefaces/primefaces/files/3406631/PF_ticket.zip)

Temporarily I've solved the problem, by reducing the icon size of the **p:selectBooleancheckbox**.
See my *.css file and uncomment the style for **.ui-chkbox .ui-chkbox-box , .ui-chkbox-icon.ui-icon**

## 5) Sample XHTML

See zip file

## 6) Sample bean

See zip file

**Comment by melloware** [comment]
Duplicate of https://github.com/primefaces/primefaces/issues/1155

**Comment by melloware** [comment]
I use this JS function to fix my frozen column rendering.

```javascript
      /**
       * Used to fix heights of rows in frozen tables
       */
      synchronizeRowsHeight : function() {
         var $leftRows = $('.ui-datatable-frozenlayout-left').find('tr');
         var $rightRows = $('.ui-datatable-frozenlayout-right').find('tr');

         $leftRows.each(function(index) {
            var $leftRow = $(this);
            var $leftHeight = $leftRow.innerHeight();
            var $rightRow = $rightRows.eq(index);
            var $rightHeight = $rightRow.innerHeight();

            if ($rightHeight > $leftHeight) {
               $leftRow.innerHeight($rightHeight);
               var diff = $rightHeight - $leftRow.innerHeight();
               if (diff != 0)
                  $leftRow.innerHeight($rightHeight + diff);
            } else if ($rightHeight < $leftHeight) {
               $rightRow.innerHeight($leftHeight);
               var diff = $leftHeight - $rightRow.innerHeight();
               if (diff != 0)
                  $rightRow.innerHeight($leftHeight + diff);
            }
         })

         // fire a resize event to tell the table to repaint
         $(window).trigger('resize');
      }
```

**Comment by wlhUser** [comment]
Thanks melloware, for the JS function. 
Although I used in the meantime the approach to fix the row heights, like described in #1155, I will try your approach as well.
